i8_terminal/service_result/service_result.py

```diff
@@ -43,8 +43,10 @@ def to_plot(self) -> Any:
     def to_xlsx(self, path: str, formatter: Optional[str] = None, styler: Optional[str] = None) -> Any:
         pass
 
-    def to_csv(self, path: str) -> Any:
-        pass
+    def to_csv(self, path: str) -> None:
+        df = self._df.copy()
+        df = self._format_df(df, format="default")
```

Comment:
format should be a parameter of the `to_csv` method with a default value of `raw`